align_system/prompt_engineering/kdma_descriptions.yml

```diff
@@ -39,3 +39,40 @@ QualityOfLife:
   valid_scores:
     values: [1,3,7,9]
   relevant_structured_character_info: [demographics.age, injuries, vitals]
+
+# NAACL24 paper dataset
```

Comment:
If possible I would like to try splitting these out into a separate file (maybe something like `naacl_kdma_descriptions.yml`), then they could be specified in the `inference_kwargs` for the config (I'll point it out in another comment).  Only because I don't want these entries to somehow interfere with our existing ADM's that use this file (not certain it would, but I think it makes sense to separate these).